src/lib/app-provider/env.ts

```diff
@@ -91,6 +91,7 @@ export enum CELATONE_QUERY_KEYS {
   TXS = "CELATONE_QUERY_TXS",
   TXS_BY_BLOCK_HEIGHT_PAGINATION = "CELATONE_QUERY_TXS_BY_BLOCK_HEIGHT",
   TXS_BY_BLOCK_HEIGHT_COUNT = "CELATONE_QUERY_TXS_BY_BLOCK_HEIGHT_COUNT",
+  TXS_BY_MODULES = "CELATONE_QUERY_TXS_BY_MODULES",
```

Comment:
no need to add just use `MODULE_TXS`

Reply:
also could you remove `useTxsCountByBlockHeight` and `getBlockTransactionCountByHeightQueryDocument` fns?

Reply:
Okay